### DISCUSSION TITLE
How commit unmodified file to SVN

### DISCUSSION THREAD

**Discussion by ankushmurkute** [AmpScm/AnkhSVN#32] [Q&A]
How commit unmodified file to SVN
 Using setting any property how can I do that
 -  I got following error
 - "Setting property on non-local targets needs a base revision"}

**Comment by ARMoir** [comment]
If the file itself is unmodified, Subversion will not create a new revision for it unless there is an actual versioned change to commit.

Changing a versioned SVN property is a valid way to create a change, but the property must be set on the local working-copy path, not directly against the repository URL. The error:

`Setting property on non-local targets needs a base revision`

usually means the property operation is being attempted against a repository target instead of the local working copy.

From the working copy, set or change a property on the file, for example with Subversion:

`svn propset <property-name> <value> <local-file-path>`

That property change should then appear in AnkhSVN Pending Changes and can be committed normally.

If the goal is simply to create an empty revision with no file or property changes, SVN does not support an empty commit in the same way Git does. There needs to be some versioned change, such as a property modification.
